lib/options.js

```diff
@@ -193,6 +193,12 @@ module.exports = optionator({
             alias: "v",
             type: "Boolean",
             description: "Outputs the version number"
+        },
+        {
+            option: "disable-inline-config",
```

Comment:
We should do this in the same manner as the `eslintrc` option. Name the option "inline-config", set default to `true`, then optionator will make that `no-inline-config`.